fashion-messaging-app-api/server.js

```diff
@@ -91,52 +91,58 @@ app.get('/posts', async (req, res) => {
     res.status(500).json({ message: err.message });
   }
 });
-app.get('/photos', async (req, res) => {
-  const query = req.query.search || defaultQuery;
 
-  
-  try {
-    const response = await clientAPI.photos.search({ query, per_page: PHOTOS, page: 1});
+    app.get('/photos', async (req, res) => {
+      const query = req.query.search || defaultQuery;
+      const perPage = (query === defaultQuery) ? 20 : PHOTOS; 
     
-    let fetchPromises = response.photos.map(async photo => {
-      if (cache.has(photo.src.original)) {
-        return cache.get(photo.src.original);
-      
-      } else {
-        var myHeaders = new Headers();
-        myHeaders.append("x-api-key", "7008b5da328806c39a5561c3d51339c18dbb66dcd8bbfeebca80f0a96c44c332");           
-                     
-
-        var formdata = new FormData();
-        formdata.append("image_url", photo.src.original);
-      
-        var requestOptions = {
-          method: 'POST',
-          headers: myHeaders,
-          body: formdata,
-        };
-
-        try {
-          const fetchResponse = await fetch('https://cloudapi.lykdat.com/v1/detection/items', requestOptions);
-          if (fetchResponse.status === 400) {
-            throw new Error('Image size too large for the API.');
+      try {
+        const response = await clientAPI.photos.search({ query, per_page: perPage, page: 1});
+        let fetchPromises = response.photos.map(async photo => {
+          if (cache.has(photo.src.original)) {
+            return cache.get(photo.src.original);
+            
+          } else if (query !== defaultQuery) { 
```

Comment:
The block of code under this helps me to exclude the defaultQuery from the fetch requests that filters fashion from non-fashion